server/sockets.js

```diff
@@ -2,8 +2,8 @@ var iolib = require('socket.io')
 	, log = require("./log.js").log
 	, BoardData = require("./boardData.js").BoardData;
 
-var MAX_EMIT_COUNT = 64; // Maximum number of draw operations before getting banned
-var MAX_EMIT_COUNT_PERIOD = 5000; // Duration (in ms) after which the emit count is reset
+var MAX_EMIT_COUNT = 200; // Maximum number of draw operations before getting banned
+var MAX_EMIT_COUNT_PERIOD = 1000; // Duration (in ms) after which the emit count is reset
```

Comment:
200 messages per second per user is a lot to handle for a server ! I think a dozen cursor updates per second are okay...

Reply:
We should let this be configured in `configuration.js`. The cursor alone can send up to 60 messages per second with the currently used tool adding the same (assuming we throttle them as well). 
I have not been able to perform stress tests with a lot of connected devices so I don't know how large the performance impact is.
@rdbeach you currently run your server with a limit of 2000 messages per second per user and with no throttling on the cursor updates. How much impact is it actually?

Reply:
I have the luxury of just having a few people on my board at a time, so I admit I haven't as yet put a lot of effort into efficiency/stress testing.

That being said, I just updated the pointer with your suggestion of throttling. Currently 20 updates per second for the pointer. Also, I changed it so that when you are drawing there are no separate pointer messages. 1ad204b4e28bade65c9296a904f316a25b6c396c

@lovasoa how does the board hold up with several users/several boards running simultaneously?

Also, one issue I had with the original board was that the pencil was throttled too much to my liking, so that the reproduction on the other persons board was noticeably different in some cases. 

`
if (curLineId !== "" && performance.now() - lastTime > 70) {\n\r
	Tools.drawAndSend(new PointMessage(x, y));
	lastTime = performance.now();
}
`
So I reduced the throttle time, but then I had to raise the MAX_EMIT_COUNT to prevent getting banned.

One more issue with the code above is that sometimes the final (x,y) position before the mouseup  event is skipped or lost, and since many people do a quick stroke before they lift their pencil when writing, some of the character and clarity of the letters is lost. To deal with this, I made sure that the final mouse position was sent, overriding the throttle.  Still, at 70ms, handwriting will not be reproduced faithfully, reducing this to 20-30 seems to do the trick, of course, at the expense of bandwidth.

Reply:
I have implemented your change to the pencil tool in #47.
Not sending the cursor updates when using a tool is a great idea, I will have a look at your commit and add it to this PR.

Reply:
Ideally, MAX_EMIT_COUNT would be passed to the client, which could use it to compute its own optimal throttling. For the pen, these are very interesting improvements !

Reply:
Batch processing is definitely preferable to dropping them, especially since the user would never notice that others can't see them.

I'm wondering whether that is still in the scope of this PR or if we should split that into another one and finish the cursor without the more intelligent throttling for now?

Reply:
Yes, we should probably do that in a separate PR. This should not be too hard, since there is already a batching mechanism, with `_children`